.github/workflows/pushtest.yml

```diff
@@ -33,24 +33,24 @@ jobs:
     - name: Install the built wheel and test usage (UNIX)
       if: matrix.operating-system == 'ubuntu-latest' || matrix.operating-system == 'macOS-latest'
       run: |
-        pip install --ignore-installed dist/imageinary-*-py3-none-any.whl
+        pip install --ignore-installed dist/nvidia_imageinary-*-py3-none-any.whl
```

Comment:
I'm a novice here, but are these supposed to be "nvidia_imageinary" rather than "nvidia-imageinary" like nearly everything else in this PR?

Reply:
Very good catch, but this is actually the expected format, which is silly. The package name is `nvidia-imageinary`, but the wheel that is built is called `nvidia_imageinary-1.0.1-py3-none-any.whl`. So, if you are installing directly from the file, you need the first underscore. After that, you will install using the dash in the name, as will be the case when downloading from the index. I don't like it either, but that's the Python setuptools convention here.

Reply:
Superpower - ask dumb questions :heavy_check_mark:

Reply:
Criminally underrated superpower.